### ISSUE TITLE
Lucide font urls lead to 404

### ISSUE THREAD

**Issue by damntourists** [juliettef/IconFontCppHeaders#44]
It seems the urls listed in this repository for Lucide are broken for the latest release:
https://unpkg.com/lucide-static@0.407.0/font/lucide.ttf
https://unpkg.com/lucide-static@0.407.0/font/lucide.css

I can access these from the alt url listed further down in the generate script
```
class FontLC( FontKI ):               # Lucide
    font_name = 'Lucide'
    font_abbr = 'LC'
    font_data_prefix = '.icon-'
    font_data = 'https://unpkg.com/lucide-static@latest/font/lucide.css'    # alt 'https://cdn.jsdelivr.net/npm/lucide-static@latest/font/lucide.css'
    ttfs = [[ font_abbr, 'lucide.ttf', 'https://unpkg.com/lucide-static@latest/font/lucide.ttf' ]]  # alt 'https://cdn.jsdelivr.net/npm/lucide-static@latest/font/lucide.ttf'
```
These alt urls are in comments though, so those urls are never tried it seems. 

I'm curious if this is a common occurrence? I use this repository as a submodule for another project and part of my process is embedding the font files into the project on build, however, the Lucide font seems to be missing. It seems in its current state it would likely cause your script to fail to generate the lucide fonts since the alt urls are never used in code. 

If you go to the urls directly that the script is trying to use, they produce the following:
```
Cannot find "/font/lucide.ttf" in lucide-static@0.407.0
Cannot find "/font/lucide.css" in lucide-static@0.407.0
```

Would it be better to fix this on a case by case basis when it crops up, or would it make sense to add in some additional functionality to consider alternative urls should the first url fail? If you like I could open up a PR for this later on when I have a free moment.

**Comment by juliettef** [comment]
Thanks for your suggestion. The issue of Lucide downloads has already been addressed in related [PR #41](https://github.com/juliettef/IconFontCppHeaders/pull/41).

The generator script is only meant to be executed manually and occasionally. Its output is updated regularly in this repository so you can download it, see `IconsLucide.*`. Note that obtaining the `*.ttf` files is not in the remit of this project.

**Comment by damntourists** [comment]
It looks like there has been a 0.408.0 version published since I created this ticket which works as expected. If you manually follow the url you mention, it resolves to a specific version, which was broken for 0.407.0.

Thanks for your time!
